modules/op-results-opc-pipelinerunresults.adoc

```diff
@@ -0,0 +1,114 @@
+// This module is included in the following assembly:
+//
+// * records/using-tekton-results-for-openshift-pipelines-observability.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="results-opc-pipelinerunresults_{context}"]
+= Viewing result information for a pipeline run
+
+[role="_abstract"]
+You can use the `opc` utility to view a description of when and how a pipeline run completed, a full manifest for the pipeline run, and any logs that the pipeline run produced.
+
+.Prerequisites
+
+* You installed the `opc` utility.
+* You configured the `opc` utility to query results from {tekton-results} by pipeline run and task run names.
+* You have the name or UUID of the pipeline run. You can use the `ocp results list pipelineruns` commands to view names and UUIDs of pipeline runs for which results are available.
+
+.Procedure
+
+* Use any of the following commands to view the result information for a pipeline run:
+** To view a description of when and how the pipeline run completed, enter the following command:
++
+[source,terminal]
+----
+$ opc results pipelinerun describe -n <namespace_name> <pipelinerun_name>
+----
++
+Alternatively, you can use the pipeline run UUID instead of the name:
++
+[source,terminal]
+----
+$ opc results pipelinerun describe -n <namespace_name> --uid <pipelinerun_uuid>
+----
```

Comment:
This is nice to include, but I worry over whether the expansiveness threatens conciseness.

Reply:
Sometimes names are VERY similar and easy to confuse, so people prefer UUIDs